newIDE/app/src/ObjectEditor/Editors/SpriteEditor/PointsEditor/PointsPreview.js

```diff
@@ -230,30 +269,34 @@ const PointsPreview = (props: Props) => {
     overflow: 'visible',
   };
 
+  const nonDefaultPoints = pointsContainer.getAllNonDefaultPoints();
+  const backgroundPointNames = [
+    ...mapVector(nonDefaultPoints, (point, i) => point.getName()),
+    'Origin',
+    'Center',
+  ].filter(name => name !== selectedPointName && name !== highlightedPointName);
+
   return (
     <div
       style={styles.containerStyle}
       onPointerMove={onPointerMove}
       onPointerUp={onEndDragPoint}
     >
       <svg style={svgStyle} ref={svgRef}>
-        {points}
-        {renderPoint(
-          'Origin',
-          originPoint.getX() * imageZoomFactor,
-          originPoint.getY() * imageZoomFactor,
-          pointKindIdentifiers.ORIGIN,
-          originPoint
-        )}
-        {renderPoint(
-          'Center',
-          (automaticCenterPosition ? imageWidth / 2 : centerPoint.getX()) *
-            imageZoomFactor,
-          (automaticCenterPosition ? imageHeight / 2 : centerPoint.getY()) *
-            imageZoomFactor,
-          pointKindIdentifiers.CENTER,
-          centerPoint
+        {/* Z index does not apply in SVG. To display selected and highlighted points
+        above the other points, they must be rendered below the other ones. */}
```

Comment:
Because they are above z-orderly speaking.
```suggestion
        above the other points, they must be rendered after the other ones. */}
```